policy/modules/system/systemd.if

```diff
@@ -40,7 +40,6 @@ template(`systemd_role_template',`
 	#
 	# Declarations
 	#
-	type systemd_user_session_type, systemd_log_parse_env_type;
```

Comment:
If the lint is coming up as this being an error, then the linter has a bug. See its use in `systemd.te`.

Reply:
It's the lint from github - the automated lint check

Reply:
Probably the correction is:

@@ -27,7 +27,7 @@
 #
 template(`systemd_role_template',`
        gen_require(`
-               attribute systemd_user_session_type, systemd_log_parse_env_type;
+               attribute systemd_user_session_type;
                attribute systemd_user_activated_sock_file_type, systemd_user_unix_stream_activated_socket_type;
                type $1_systemd_t;
                type systemd_run_exec_t, systemd_analyze_exec_t;

Reply:
And then I decided to remove the type declaration which seems uneeded and reedundant with the attribute declaration above...

Reply:
It's not being used as a type in that interface. The use case is to use the attribute in "typeattribute" rules in other interfaces such as the one that I added in that same file.

Reply:
And since you referred to systemd.te, then that confirms that it is being declared only as an attribute even there. So, the latter change that I introduced, although possibly not stricltly necessary to shut up the linter, does make sense, improves code clarity to me.

Reply:
Yes, I missed that it said type. The correction should be to make it an attribute requirement. I don't think having the interface significantly improves readability. The worse thing is that it makes it look like other domains could be systemd user sessions, and that's not true.

Reply:
So, do you sugget removing the interface ?